jsonify.js

```diff
@@ -72,14 +67,18 @@ var lf_processFile = function(p_strFile) {
         l_strFileReplace = l_strFileReplace.replace(/\n/, ' ').replace(/\r/, ' ').replace(/\"/g, "\\\"");
 
         l_strContent = l_strContent.replace(/@file:.+\.(html|js)/, l_strFileReplace);
-
     }
     if ('' != l_strCompanyGuid){
         l_strContent = l_strContent.replace(/"companyId"\s*:\s*""/,'"companyId" : "' + l_strCompanyGuid + '"');
     }
     l_strContent = l_strContent.replace(/\[/,"").replace(/]\s*$/, "").replace(/\r\n/g,' ').replace(/\n/g, ' ');
     return l_strContent;
```

Comment:
Does `l_strContent` contain the final file contents that were processed and returned?